05_iigo/index.tex

```diff
@@ -347,50 +348,67 @@ \section{Accountability Cycle}
 
 \subsection{Transfer-of-power}
 \label{subsec:transfer-of-power}
+
+For the scope of this project we chose only one system of power transfer for the islands to utilise, and it is an election. The islands that hold institutional power are the decision group of the archipellago. They decide on taxes, allocations and sanctions. By holding an election for the institutional roles, the islands are not directly included in the decision group, but they do participate in deciding who will occupy these roles, who makes the aforementioned decisions. An election also opens up another avenue for opinion formation to have an effect. 
+\begin{itemize}
+    \item Each role has the \emph{power} to call an election vote and declare the winner (see Figure~ \ref{fig:cycles_in_IIGO} for the power transfer cycle). 
+\end{itemize}
+
+We note that:
 \begin{enumerate}
-    \item The Speaker conducts a vote for the election of the new Judge.
-    \item The Judge conducts a vote for the election of the new President.
-    \item The President conducts a vote for the election of the new Speaker.
+    \item The Speaker conducts an election for the election of the new Judge.
+    \item The Judge conducts an election for the election of the new President.
+    \item The President conducts an election for the election of the new Speaker.
 \end{enumerate}
 Refer to the Figure~ \ref{fig:cycles_in_IIGO} for further clarification about the transfer-of-power cycle.
 
-\begin{rule_IIGO}
-   The role $X$ has the \emph{obligation} to conduct a vote for the election of $Y$ at the end of the term (see Definition~\ref{def:term}) if and only if the role $X$ is accountable to the role $Y$. 
-\end{rule_IIGO}.
+We introcude a \emph{term} length to futher spread out the diversity of the decision group. If the Rule~\ref{rule:roles_must_hold_election} is in play, the roles are obliged to hold an election every N turns. To reduce the scope of the coursework, the term length is defined as a configuration parameter. Thus we reduce the complexity of reasoning the agents have to do with reagard to the transfer-of-power of roles.
+
+\begin{rule_IIGO} \label{rule:roles_must_hold_election}
```

Comment:
announcement..
typo!
finish by a DOT. at the end

Reply:
`$N$` 

 To reduce the scope of the coursework,  ---> delete.
 
 **The term length for each IIGO role is defined as a configuration parameter.**   this can stay
 
 Thus we reduce the complexity of reasoning the agents have to do with reagard to the transfer-of-power of roles.
 ----> delete or REVISE cuz I don't understand

Reply:
Its important to justify design decisions hence I disagree with the removal of to reduce the scope of the coursework

Reply:
then revise the sentence plz..
what is the compelling design decision here?

Reply:
what is the compelling design decision here? I don't understand this question

Reply:
the design choice cannot be understood from this wording ....
is what I'm saying